### ISSUE TITLE
Upgrade from C130 -> C131 on macOS via 1.73.x - Nightly (Checking C131)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#41841]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by LaurenWags** [comment]
Tests above verified with

```
Brave	1.73.39 Chromium: 131.0.6778.14 (Official Build) nightly (arm64) 
Revision	6efd8010cd2636a33b10422509349592bcbce006
OS	macOS Version 15.1 (Build 24B82)
```

**Comment by GeetaSarvadnya** [comment]
Upgrade checks (using arm64 .dmg):

```

```

Note, need to be on internal VPN to do the below upgrade checks.

<details>
<summary>C131 --> C131 on-demand upgrade check - PASSED</summary>

1. Install `C130` PR build (arm64 builds available via https://bravesoftware.slack.com/archives/CU5UXM3EC/p1727798586109619?thread_ts=1727798559.900959&cid=CU5UXM3EC)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)

<img width="1218" alt="Screenshot 2024-10-03 at 6 51 03 PM" src="https://github.com/user-attachments/assets/4a017650-d36d-4706-9b3c-2b330c2bff2b">

```
[1392:259:1003/184942.523472:INFO:sparkle_glue.mm(346)] brave update: did not find update
```
5. Close `Brave`
6. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
7. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
8. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```
[1708:259:1003/185346.530402:INFO:sparkle_glue.mm(327)] brave update: did finish loading appcast
[1708:259:1003/185346.532386:INFO:sparkle_glue.mm(333)] brave update: did find valid update with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[1708:259:1003/185346.534619:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[1708:259:1003/185408.756128:INFO:sparkle_glue.mm(390)] brave update: will install update on quit with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
```

9. Confirm you are prompted to "Relaunch" to get update (note, if you click "Relaunch" button you will relaunch and could potentially pick up a newer Nightly that is on the previous Chromium version. Closing and then relaunching via the terminal will not update you again)

<img width="1206" alt="Screenshot 2024-10-03 at 6 58 21 PM" src="https://github.com/user-attachments/assets/9e5ec39a-3895-43b5-aa1f-77b397a93e76">


Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------
<img width="1209" alt="Screenshot 2024-10-03 at 6 59 47 PM" src="https://github.com/user-attachments/assets/1df8985c-7281-4e4c-a041-2539c940421c"> | <img width="1207" alt="Screenshot 2024-10-03 at 7 00 07 PM" src="https://github.com/user-attachments/assets/e6525ace-36fd-43eb-9984-8b6d51a74393">


Confirmed deltas:

**`Example`** | **`Example`**
----------------|----------------
<img width="1433" alt="Screenshot 2024-10-03 at 7 20 09 PM" src="https://github.com/user-attachments/assets/fc3a2fc4-ea21-4caa-85b2-9e1e98c631ab"> | <img width="1439" alt="Screenshot 2024-10-03 at 7 12 48 PM" src="https://github.com/user-attachments/assets/6ca71362-6f22-4722-90d2-ec3799066bea">



</details>

<details>
<summary>C130 --> C131 on-demand upgrade check - PASSED</summary>

1. Install `C129` build via https://github.com/brave/brave-browser/releases/tag/v1.72.48
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)

<img width="1213" alt="Screenshot 2024-10-03 at 6 23 02 PM" src="https://github.com/user-attachments/assets/f227d945-7073-4624-ac11-7ff828df17d3">

```
[1089:259:1003/182157.635315:INFO:sparkle_glue.mm(346)] brave update: did not find update
```

4. Close `Brave`
5. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
6. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```
[1203:259:1003/182535.981486:INFO:sparkle_glue.mm(327)] brave update: did finish loading appcast
[1203:259:1003/182535.983308:INFO:sparkle_glue.mm(333)] brave update: did find valid update with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[1203:259:1003/182535.985032:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[1203:259:1003/182604.000216:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[1203:259:1003/182909.848845:INFO:sparkle_glue.mm(390)] brave update: will install update on quit with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
```

7. Confirm you are prompted to "Relaunch" to get update (see note in first case about this button)

<img width="1212" alt="Screenshot 2024-10-03 at 6 29 38 PM" src="https://github.com/user-attachments/assets/6ef76743-6377-43f5-8621-f99d411c8b76">


Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------
<img width="1214" alt="Screenshot 2024-10-03 at 6 41 34 PM" src="https://github.com/user-attachments/assets/85b0ba4d-493f-4949-9b0d-48f3f2ace6c8"> | <img width="1209" alt="Screenshot 2024-10-03 at 6 43 39 PM" src="https://github.com/user-attachments/assets/1e7a8b40-dc7d-4acf-809e-54cd2cb2d4ad">


</details>

<details>
<summary>C131 --> C131 background upgrade check - PASSED</summary>

1. Install `C130` PR build (arm64 builds available via https://bravesoftware.slack.com/archives/CU5UXM3EC/p1727798586109619?thread_ts=1727798559.900959&cid=CU5UXM3EC)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)

<img width="1213" alt="Screenshot 2024-10-03 at 7 36 27 PM" src="https://github.com/user-attachments/assets/deeb7558-c7f0-427d-a8e9-ff55f5106cab">

```
[2843:259:1003/193624.342999:INFO:sparkle_glue.mm(346)] brave update: did not find update
```

4. Close browser
5. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
6. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
7. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
8. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
9. DO NOT navigate to `brave://settings/help`
10. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```
[925:259:1003/195700.150966:INFO:sparkle_glue.mm(327)] brave update: did finish loading appcast
[925:259:1003/195700.152179:INFO:sparkle_glue.mm(333)] brave update: did find valid update with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[925:259:1003/195700.152609:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[925:259:1003/195723.130767:INFO:sparkle_glue.mm(390)] brave update: will install update on quit with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)

```

11. Close and restart the browser via terminal with same command as step 8
12. Check version is updated under `brave://version`

<img width="1213" alt="Screenshot 2024-10-03 at 8 05 28 PM" src="https://github.com/user-attachments/assets/66127d3f-aba1-4094-ab3c-bac4a3f90a9d">


13. Confirmed deltas:

Example | Example
--- | ---
<img width="1440" alt="Screenshot 2024-10-03 at 8 07 09 PM" src="https://github.com/user-attachments/assets/c472dc39-54a8-40e2-8af3-11586b10f257"> | <img width="1434" alt="Screenshot 2024-10-03 at 8 07 46 PM" src="https://github.com/user-attachments/assets/d0043255-201c-4b0f-a0de-e0d55cc42d0a">



</details>

<details>
<summary>C130 --> C131 background upgrade check - PASSED</summary>

1. Install `C129` build via https://github.com/brave/brave-browser/releases/tag/v1.72.48
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)
5. Close browser
6. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
7. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
8. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
10. DO NOT navigate to `brave://settings/help`
11. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```
[660:259:1003/203348.212002:INFO:sparkle_glue.mm(327)] brave update: did finish loading appcast
[660:259:1003/203348.214014:INFO:sparkle_glue.mm(333)] brave update: did find valid update with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/203348.214873:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/203434.257303:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/204005.988968:INFO:sparkle_glue.mm(390)] brave update: will install update on quit with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)


```

12. Close and restart the browser via terminal with same command as step 8
13. Check version is updated under `brave://version`

<img width="1229" alt="Screenshot 2024-10-03 at 8 42 17 PM" src="https://github.com/user-attachments/assets/6c2c9fa2-bacd-47f6-a2ac-56192a6285b6">


</details>

**Comment by LaurenWags** [comment]
Upgrade checks (using arm64 .dmg):

```

```

Note, need to be on internal VPN to do the below upgrade checks.

<details>
<summary>C131 --> C131 background upgrade check - InProgress</summary>

1. Install `C130` PR build (arm64 builds available via https://bravesoftware.slack.com/archives/CU5UXM3EC/p1727798586109619?thread_ts=1727798559.900959&cid=CU5UXM3EC)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)

```

```

4. Close browser
5. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
6. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
7. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
8. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
9. DO NOT navigate to `brave://settings/help`
10. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```

```

11. Close and restart the browser via terminal with same command as step 8
12. Check version is updated under `brave://version`

SS

13. Confirmed deltas:

Example | Example
--- | ---

</details>

<details>
<summary>C130 --> C131 background upgrade check - TBD</summary>

1. Install `C130` build via https://github.com/brave/brave-browser/releases/tag/v1.73.38
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)
5. Close browser
6. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
7. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
8. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly-arm64/appcast.xml`
10. DO NOT navigate to `brave://settings/help`
11. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```
[660:259:1003/203348.212002:INFO:sparkle_glue.mm(327)] brave update: did finish loading appcast
[660:259:1003/203348.214014:INFO:sparkle_glue.mm(333)] brave update: did find valid update with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/203348.214873:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/203434.257303:INFO:sparkle_glue.mm(354)] brave update: willDownloadUpdate with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)
[660:259:1003/204005.988968:INFO:sparkle_glue.mm(390)] brave update: will install update on quit with AppcastItem(Date: Tue, 01 Oct 2024 16:02:26 +0000, Version: 1.72.51.0)


```

12. Close and restart the browser via terminal with same command as step 8
13. Check version is updated under `brave://version`

<img width="1229" alt="Screenshot 2024-10-03 at 8 42 17 PM" src="https://github.com/user-attachments/assets/6c2c9fa2-bacd-47f6-a2ac-56192a6285b6">


</details>

**Comment by LaurenWags** [comment]
Upgrade checks (using x64 .dmg):

```

```

and

```

```

Encountered https://github.com/brave/brave-browser/issues/41360 while testing.

Note, need to be on internal VPN to do the below upgrade checks.

<details>
<summary>C131 --> C131 on-demand upgrade check - InProgress</summary>

1. Install `C131` PR build (x64 builds available via TBD)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)
4. Close `Brave`
5. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
6. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly/appcast.xml`
7. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```

```

8. Confirm you are prompted to "Relaunch" to get update (note, if you click "Relaunch" button you will relaunch and could potentially pick up a newer Nightly that is on the previous Chromium version. Closing and then relaunching via the terminal will not update you again)

SS

Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------

Check deltas:

**`Example`** | **`Example`**
----------------|----------------

</details>

<details>
<summary>C130 --> C131 on-demand upgrade check - TBD</summary>

1. Install `C130` build via https://github.com/brave/brave-browser/releases/tag/vTBD
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)
4. Close `Brave`
5. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly/appcast.xml`
6. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```

```

7. Confirm you are prompted to "Relaunch" to get update (see note in first case about this button)

SS

Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------

</details>

<details>
<summary>C131 --> C131 background upgrade check - TBD</summary>

1. Install `C131` PR build (arm64 builds available via TBD)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)
4. Close browser
5. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
6. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
7. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
8. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly/appcast.xml`
9. DO NOT navigate to `brave://settings/help`
10. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```

```

11. Close and restart the browser via terminal with same command as step 8
12. Check version is updated under `brave://version`

SS

13. Confirmed deltas:

Example | Example
--- | ---

</details>

<details>
<summary>C130 --> C131 background upgrade check - TBD</summary>

1. Install `C130` build via https://github.com/brave/brave-browser/releases/tag/vTBD
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but be sure to close this tab prior to closing the browser)
4. Close browser
5. Delete the following:
- `com.brave.Browser.nightly` folder from `~/Library/Caches`
- `com.brave.Browser.nightly.plist` file from `~/Library/Preferences`
- Alternatively you can probably advance your computer clock ~4 hours, I am fairly certain deleting at least one of the above files removes the timer check for background updates.
6. Restart your computer if you deleted files (if you just advanced computer clock this step isn't necessary)
7. After computer restart, Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly/appcast.xml`
9. DO NOT navigate to `brave://settings/help`
10. Watch the logs and see that an update is found (be sure you have all of the below before proceeding)

```

```

11. Close and restart the browser via terminal with same command as step 7
12. Check version is updated under `brave://version`

SS

</details>

**Comment by LaurenWags** [comment]
Upgrade checks (using x64 .dmg):

```

```

and

```

```

Note, need to be on internal VPN to do the below upgrade checks.

<details>
<summary>C132 --> C132 on-demand upgrade check - InProgress</summary>

1. Install `C131` PR build (see https://bravesoftware.slack.com/archives/CU5UXM3EC/p1732631161761129)
2. Launch via terminal as follows: `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nothanks/appcast.xml` (this is so you don't pick up any newer Nightly versions that have already been uploaded to sparkle)
3. Watch logs, confirm no update found (can also visit `brave://settings/help` but close this tab prior to closing the browser)
4. Close `Brave`
5. Note, if you want to check deltas, at this point open something like Fiddler Everywhere before continuing.
6. Relaunch with `/Applications/Brave\ Browser\ Nightly.app/Contents/MacOS/Brave\ Browser\ Nightly --args --enable-logging=stderr --update-feed-url=https://updates-panel.brave.com/sparkle/Brave-Browser/nightly/appcast.xml`
7. Navigate to `brave://settings/help` and ensured that an update has been found/being applied

```

```

8. Confirm you are prompted to "Relaunch" to get update (note, if you click "Relaunch" button you will relaunch and could potentially pick up a newer Nightly that is on the previous Chromium version. Closing and then relaunching via the terminal will not update you again)

SS

Once relaunched, ensured the correct version was installed as per the following:

**`Example`** | **`Example`**
----------------|----------------

Check deltas:

**`Example`** | **`Example`**
----------------|----------------

</details>
